docker/build-tools/Dockerfile

```diff
@@ -270,8 +270,11 @@ RUN set -eux; \
     esac; \
     \
     wget "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/${GCLOUD_TAR_FILE}"; \
-    tar -xzvf ./${GCLOUD_TAR_FILE} -C ${OUTDIR}/usr/local && rm ${GCLOUD_TAR_FILE}; \
-    ${OUTDIR}/usr/local/google-cloud-sdk/bin/gcloud components install beta kpt --quiet; \
+    tar -xzf ./${GCLOUD_TAR_FILE} -C ${OUTDIR}/usr/local && rm ${GCLOUD_TAR_FILE}; \
```

Comment:
Should we fetch kpt from GitHub? https://github.com/GoogleContainerTools/kpt/releases it seems to be available here.